README.md

```diff
@@ -27,3 +25,5 @@ Náhledový server vyžaduje autentizaci, aby ho nemohl používat kdokoliv mimo
 ### forum.ohlasy.info
 
 Virtuální server u [Digital Ocean](https://digitalocean.com), ručně instalovaná instance [Discourse](https://www.discourse.org).
+
+![Powered by Vercel](/assets/vercel.svg?raw=true)
```

Comment:
Please make this a link that goes to https://www.vercel.com?utm_source=[ohlasy]&utm_campaign=oss

**for Vercel Sponsorship**

Reply:
Thank you, fixed!